src/sentry/static/sentry/app/views/organizationDetails/index.jsx

```diff
@@ -161,8 +183,14 @@ export default class OrganizationDetails extends Component {
   render() {
     return (
       <OrganizationContext includeSidebar useLastOrganization {...this.props}>
-        <OrganizationDetailsBody>{this.props.children}</OrganizationDetailsBody>
+        <OrganizationDetailsBody detailed={this.props.detailed}>
```

Comment:
Does `<OrganizationContext>` need `detailed`? If not, we can destructure `detailed` from props, and pass the rest of the object to `<OrganizationContext>`? That way it's a bit more clear who needs the `detailed` prop.